src/components/Landing/LandingPage.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import  "./LandingPage.css"
+import Graph from './Graph'
+import Overview from './Overview'
+import logo from '../../assets/logo.png'
+
+export default function LandingPage() {
+
+  // Just some dummy data for the graph
```

Comment:
make a separate file and import/store dummy data from there.

Reply:
done